<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VErrorHistory</class>
 <widget class="QWidget" name="VErrorHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1750</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDateEdit" name="edtStartDate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblStartDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="edtEndDate">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblEndDate">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>End Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="edtDeleteDate">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblDeleteDate">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>30</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete data olderï¼š</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>20</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>1580</x>
     <y>20</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>1711</width>
     <height>601</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ErrorDateTime</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AlarmCode</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ErrorDescription</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
